src/pages/Home.js

```diff
@@ -0,0 +1,60 @@
+import { Navbar } from "../components/Navbar";
+import { Footer } from "../components/Footer";
+import "../styles/pages/home.css"
+import { FeaturedCategories } from "../components/FeaturedCategories";
+export const Home = ()=>{
+
+    const featuredCategories = [
+    {category:"item-1",
+    title:"Fertilizers",
+    imgSrc:"https://res.cloudinary.com/dvuh4fz9d/image/upload/v1648390941/fertilizers1_hdthei.jpg"},
+    {category:"item-2",
+    title:"Pesticides",
+    imgSrc:"https://res.cloudinary.com/dvuh4fz9d/image/upload/v1648390940/pesticides1_pveijq.jpg"},
+    {category:"item-3",
+    title:"Farming Tools",
+    imgSrc:"https://res.cloudinary.com/dvuh4fz9d/image/upload/v1648390940/tools_emqkuu.jpg"},
+    {category:"item-4",
+    title:"Seeds",
+    imgSrc:"https://res.cloudinary.com/dvuh4fz9d/image/upload/v1648390941/seeds_tqx9ub.jpg"},
+    {category:"item-5",
+    title:"Safety Wearables",
+    imgSrc:"https://res.cloudinary.com/dvuh4fz9d/image/upload/v1648395553/farmer-spraying-rice_720x480_p9nhyb.webp"},
+    {category:"item-6",
+    title:"Electronic Devices",
+    imgSrc:"https://res.cloudinary.com/dvuh4fz9d/image/upload/v1648397135/probeInSoil_1_cropped_s81abe.jpg"},
+]
+
```

Comment:
Yo can load the data from API

Reply:
Ok Thanks, suggested changes made in new commit